independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json

```diff
@@ -0,0 +1,500 @@
+                "type": "object",
+                "properties": {
+                  "homePageUrl": {
+                    "type": "string",
+                    "description": "Home page or landing page of the application"
+                  },
+                  "logoutUrl": {
+                    "type": "string",
+                    "description": "Specifies the URL that will be used by Microsoft's authorization service to logout an user using front-channel, back-channel or SAML logout protocols"
+                  },
+                  "redirectUris": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "description": "Specifies the URLs where user tokens are sent for sign-in, or the redirect URIs where OAuth 2.0 authorization codes and access tokens are sent"
+                  },
+                  "implicitGrantSettings": {
+                    "type": "object",
+                    "properties": {
+                      "enableAccessTokenIssuance": {
+                        "type": "boolean",
+                        "description": "Specifies whether this web application can request an ID token using the OAuth 2.0 implicit flow"
+                      },
+                      "enableIdTokenIssuance": {
+                        "type": "boolean",
+                        "description": "Specifies whether this web application can request an access token using the OAuth 2.0 implicit flow"
+                      }
+                    },
+                    "description": "Specifies whether this web application can request tokens using the OAuth 2.0 implicit flow"
+                  }
+                },
+                "description": "Specifies settings for a web application"
+              },
+              "spa": {
+                "type": "object",
+                "properties": {
+                  "redirectUris": {
+                    "type": "array",
+                    "items": {},
+                    "description": "Specifies the URLs where user tokens are sent for sign-in, or the redirect URIs where OAuth 2.0 authorization codes and access tokens are sent"
+                  }
+                },
+                "description": "Specifies settings for a single-page application, including sign out URLs and redirect URIs for authorization codes and access tokens"
+              }
+            }
+          },
+          "description": "Represents an application. Any application that outsources authentication to Azure Active Directory (Azure AD) must be registered in a directory"
+        }
+      }
+    }
+  },
+  "parameters": {
+  },
+  "securityDefinitions": {
+    "undefined": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+      "scopes": {}

```

Comment:
Hi @paulculmsee I hope you are doing well.
Apologies for the delay in the process.
Can you please add the scopes that you are targeting the Graph API for, and be more specific?
Thank you very much.